Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionCodeGenerator.cs

```diff
@@ -41,6 +42,54 @@ public static class InputActionCodeGenerator
     {
         private const int kSpacesPerIndentLevel = 4;
 
+        private const string kClassExample = @"using namespace UnityEngine;
+using UnityEngine.InputSystem;
+
+// Example of using an InputActionMap named ""Player"" from a MonoBehavior implementing callback interface.
```

Comment:
Should be MonoBehaviour in the comment, super minor

Reply:
Thanks, its these kind of things I was sure I was going to mess up, will update

Reply:
Fixed in https://github.com/Unity-Technologies/InputSystem/pull/2045/commits/2658f2972f7da3741e6a020381f3d7daeac32e08